persist/schema/types.go

```diff
@@ -27,12 +27,14 @@ const MajorVersion = 1
 
 // IndexInfo stores metadata information about block filesets
 type IndexInfo struct {
-	Start        int64
+	BlockStart   int64
```

Comment:
could you maintain ordering of the fields here 1-1 with how you're serialising

Reply:
yeah I was trying to keep them conceptually ordered because the serial ordering is the result of when we added new fields, but I'll keep them in serial order its probably better that way

Reply:
done

Reply:
Yeah the ordering is mainly for sanity when people are looking at the two files at the same time. Makes it much easier to compare line by line rather than all at once.